Model-Docs/example-model.yaml

```diff
@@ -41,6 +74,9 @@ input_files:
             ontology: AGROVOC
             uri: http://aims.fao.org/aos/agrovoc/c_6161
         # Optional: specify arbitrary metadata, such as units
+        
+        #Why not have just "units"??. 
```

Comment:
This is how DCAT handles units. `units` are within the arbitrary `metadata` allowed for a variable.

Reply:
I think it's a bit redundant, but if you need to add it, then i won't fight against it.